wolfssl/wolfcrypt/ecc.h

```diff
@@ -504,6 +504,9 @@ struct ecc_key {
 #if defined(PLUTON_CRYPTO_ECC) || defined(WOLF_CRYPTO_CB)
     int devId;
 #endif
+#if defined(WOLF_PRIVATE_KEY_ID) || defined(WOLFSSL_ASYNC_CRYPT)
```

Comment:
Please only wrap with `HAVE_PKCS11`

Reply:
pahole shows that in our configuration, on x86-64, there is a 4 byte hole after devId, so it seems a good place to put this flag.

Reply:
On the other hand, I guess the field is only used to tell wc_InitRsaKey_ex and wc_ecc_init_ex to not init the asyncDev device, right?

Reply:
@fencekicker, I can move it there. Yes, it would only be used in the case you described

Reply:
@fencekicker, yes it would only be used in the case you described